docs/distributed-tracing-guide.asciidoc

```diff
@@ -23,22 +26,23 @@ var apm = initApm({
 })
 ----
 
-
 This effectively tells the agent to add the distributed tracing HTTP header (`elastic-apm-traceparent`) 
 to requests made to `https://api.example.com`.
 
+To read more about cross-origin requests and why this process is necessary,
+please see the MDN page on https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS[Cross-Origin Resource Sharing].
+
 [float]
 [[server-configuration]]
 === Server configuration
```

Comment:
I'm confused about this heading. It seems the content both before and after the heading relate to CORS?

Reply:
The part before this is related to what needs to be done on the frontend side (e.g. configuring the agent), but the server configuration refers to what needs to be done in the backend side. They are part of the CORS configuration just different places.

Reply:
Got it, thank you!